includes/application.php

```diff
@@ -7,4 +8,18 @@
 	require_once($APPLICATION_DIR.'/includes/settings.php');
 	require_once($APPLICATION_DIR.'/includes/db_connection.php');
 	require_once($APPLICATION_DIR.'/vendor/autoload.php');
-?>
\ No newline at end of file
+
+	if (!isset($_SESSION)) {
+		session_start();
+	}
+	if (!isset($_SESSION['token'])) {
+		$csrf_token = md5(uniqid(rand(), TRUE));
+		$_SESSION['token'] = $csrf_token;
+		$_SESSION['token_time'] = time();
+	} else {
+		$csrf_token = $_SESSION['token'];
+	}
+
+	$_GET   = filter_input_array(INPUT_GET, FILTER_SANITIZE_STRING);
+	$_POST  = filter_input_array(INPUT_POST, FILTER_SANITIZE_STRING);
+?>
```

Comment:
This was already done during the day. I just have to push the code. Thanks @prathmeshranaut